test/images/serve_hostname/README.md

```diff
@@ -1,7 +1,31 @@
-serve_hostname
-==============
+## serve_hostname
 
-Util app to serve your hostname on TCP and/or UDP.  Useful for testing.
+This is a small util app to serve your hostname on TCP and/or UDP.  Useful for testing.
+
+The `serve_hostname` Makefile supports multiple architectures, which means it may cross-compile and build an docker image easily.
+Arch-specific busybox images serve as base images.
+
+If you are releasing a new version, please bump the `TAG` value in the `Makefile` before building the images.
+
+## How to release:
+
+```
+# Build for linux/amd64 (default)
+$ make push
```

Comment:
Document here that one should just run `all-container` or if it's a release; run `all-push`